packages/ocamlformat-rpc/ocamlformat-rpc.removed/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Auto-formatter for OCaml code (RPC mode)"
+description:
+  "OCamlFormat is a tool to automatically format OCaml code in a uniform style. This package defines a RPC interface to OCamlFormat"
+maintainer: ["OCamlFormat Team <ocamlformat-dev@lists.ocaml.org>"]
+authors: ["Josh Berdine <jjb@fb.com>"]
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+depends: []
+build: []
+dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
+license: ["MIT" "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"]
+messages: [
+  "DEPRECATED. OCamlFormat ships the RPC mode by default since version 0.22.4. This package is no longer necessary." {installed}
```

Comment:
```suggestion
  "The `ocamlformat` package ships the RPC mode since version 0.22.4. This package is no longer necessary." {installed}
```

Let's not shout at our users :-)

Reply:
Thanks!